modules/nf-core/mirtop/counts/main.nf

```diff
@@ -3,7 +3,7 @@ process MIRTOP_COUNTS {
     label 'process_single'
 
     conda "${moduleDir}/environment.yml"
-    container "community.wave.seqera.io/library/mirtop_pybedtools_pysam_samtools_pruned:60b8208f3dbb2910"
+    container "community.wave.seqera.io/library/mirtop_pybedtools_pysam_samtools:b9705c2683e775b8"
```

Comment:
Please also include the singularity container e.g. https://github.com/nf-core/modules/blob/8b06d86f6a82b6203f239ad409f606fdf71ec697/modules/nf-core/wget/main.nf#L6-L8